docs/reference/filter.md

```diff
@@ -262,6 +263,7 @@ Used with the `SHOW_ONLY_SERIES` variable.
 | Nvidia RTX 3080 | `3080` |
 | Nvidia RTX 3080 Ti | `3080ti` |
 | Nvidia RTX 3090 | `3090` |
+| Nvidia RTX 3090 | `4090` |
```

Comment:
```suggestion
| Nvidia RTX 4090 | `4090` |
```

Reply:
Once you update this @mrmos, we can merge this in! Thanks for your contribution :)

Reply:
Actually I'm going to go ahead and merge this then update myself.

Thank you!

Reply:
Thanks jef!